include/compose.bash

```diff
@@ -261,6 +261,15 @@ logspout:
     entrypoint: ["/bin/sh"]
     command: -c 'sleep 1; ROUTE_URIS=\$\$LOGSINK_PORT_3333_TCP /bin/logspout'
 
+mail:
+    ports:
+        - "$PRIVATE_IP:25:25"
+    environment:
+        - maildomain=mail.hortonworks.com
+        - smtp_user=admin:cloudbreak
```

Comment:
If you would like to use this as smtp server, you have to set the following env vars:
export CLOUDBREAK_SMTP_SENDER_USERNAME=admin
export CLOUDBREAK_SMTP_SENDER_PASSWORD=cloudbreak
...